docs/reference/docs/reindex.asciidoc

```diff
@@ -422,6 +422,9 @@ will finish when their sum is equal to the `total` field.
 [float]
```

Comment:
Lgtm. Can you add a link to update by query as well?
On Mar 17, 2016 11:25 AM, "Alexander Reelsen" <notifications@github.com>
wrote:

> In docs/reference/docs/reindex.asciidoc
> <https://github.com/elastic/elasticsearch/pull/17168#discussion_r56522436>
> :
>
> > @@ -422,6 +422,9 @@ will finish when their sum is equal to the `total` field.
> >  [float]
>
> true, fixed
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/elasticsearch/pull/17168/files/669caa3e0a4c03c48faffc767288030825b03e5d#r56522436>
>